backend/app/api/journal.py

```
@@ -105,6 +113,179 @@ async def parse_stt(
+        raise HTTPException(502, f"Vision 분석 실패: {type(e).__name__}: {e}")
+
+    # 농약 매칭 후처리 (STT 경로와 동일)
+    try:
+        from app.core.pesticide_matcher import enrich_with_pesticide_match
+
+        enriched_entries = []
+        for entry in result.get("entries", []):
+            try:
+                enriched_entries.append(await enrich_with_pesticide_match(db, entry))
+            except Exception:
+                enriched_entries.append(entry)
```

Ruff (0.15.12)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)